python/flashinfer/utils.py

```diff
@@ -211,17 +211,19 @@ def register_custom_op(
     device_types: Optional[Union[str, Sequence[str]]] = None,
     schema: Optional[str] = None,
 ) -> Callable:
-    if TorchVersion(torch_version) < TorchVersion("2.4"):
-        return lambda x: x
-    return torch.library.custom_op(
-        name, fn, mutates_args=mutates_args, device_types=device_types, schema=schema
-    )
+    # if TorchVersion(torch_version) < TorchVersion("2.4"):
+    #     return lambda x: x
+    # return torch.library.custom_op(
+    #     name, fn, mutates_args=mutates_args, device_types=device_types, schema=schema
+    # )
+    return lambda x: x
```

Comment:
Will add `custom_op` back after we figured out the performance degradation @abcdabcd987 .